cloudnet-modules/cloudnet-bridge/src/main/java/de/dytanic/cloudnet/ext/bridge/gomint/GoMintCloudNetPlayerInfo.java

```diff
@@ -0,0 +1,153 @@
+package de.dytanic.cloudnet.ext.bridge.gomint;
+
+import de.dytanic.cloudnet.driver.network.HostAndPort;
+import de.dytanic.cloudnet.ext.bridge.WorldPosition;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+import java.util.Locale;
+import java.util.UUID;
+
+@ToString
+@EqualsAndHashCode
+final class GoMintCloudNetPlayerInfo {
+
+    protected double health, maxHealth, saturation;
+    protected int level, ping;
```

Comment:
Codacy found an issue: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/gh/CloudNetService/CloudNet-v3/pullRequest?prid=6226234)